README.md

```diff
@@ -1,36 +1,23 @@
-# Planet API Client
+# Planet SDK for Python
 
 [![Build Status](https://travis-ci.org/planetlabs/planet-client-python.svg?branch=master)](https://travis-ci.org/planetlabs/planet-client-python)
 
-Python client library for Planet's APIs.
+[Planet](https://planet.com) Software Development Kit (SDK) for Python.
 
-The client provides access to the following Planet APIs:
-* [analytics](https://developers.planet.com/docs/analytics/)
-* [basemaps](https://developers.planet.com/docs/basemaps/) (referred to in the client as `mosaics`)
-* [data](https://developers.planet.com/docs/data/)
-* [orders](https://developers.planet.com/docs/orders/)
-
-## Installation
-
-### Prerequisites
-
-* Python version 3.7+
-
-### Install package
-
-```console
-$ pip install planet
-```
+## Documentation
 
-The [--user](https://pip.pypa.io/en/stable/user_guide/#user-installs)
-flag is highly recommended for those new to [pip](https://pip.pypa.io).
+Full documentation is available at
+[planetlabs.github.io/planet-client-python](https://planetlabs.github.io/planet-client-python/index.html)
```

Comment:
this is fine for now, but like we discussed today the SDK for Python (V2) docs will not be available on github pages until 1.0 release / V1 deprecation... once I make a decision on where dev docs will live, we should update this

Reply:
agreed. I'll change this now so the README isn't pointing to the V1 documentation, as that's *super* not helpful for V2!